servatrice/src/servatrice_database_interface.cpp

```diff
@@ -923,3 +923,31 @@ void Servatrice_DatabaseInterface::updateUsersLastLoginData(const QString &userN
         }
     }
 }
+
+QList<ServerInfo_Ban> Servatrice_DatabaseInterface::getUserBanHistory(const QString userName)
+{
+    QList<ServerInfo_Ban> results;
+    ServerInfo_Ban banDetails;
+
+    if (!checkSql())
+        return results;
+
+    QSqlQuery *query = prepareQuery("SELECT A.id_admin, A.time_from, A.minutes, A.reason, A.visible_reason, B.name AS name_admin FROM {prefix}_bans A LEFT JOIN {prefix}_users B ON A.id_admin=B.id WHERE A.user_name = :user_name");
+    query->bindValue(":user_name", userName);
+
+    if (!execSqlQuery(query))
+        return results;
```

Comment:
Log the failure reason?

Can we add the logging-failed-queries behavior directly into `execSqlQuery`? That way we can't forget